docs/gh_pages/docs/troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+# Troubleshooting Pebblo
+
+## Issued found in conda virtual env.
+### 1. OSError: cannot load library 'pango-1.0-0'
```

Comment:
This is common troubleshooting guide for `Installation`, `Development Environment` and `Pebblo Safe DataLoader for Langchain` sections of this document. IMO, So there should be either some identifier to notify that this particular troubleshooting issue belongs to what section. This will be easy for end user.